.zuul.yaml

```diff
@@ -0,0 +1,216 @@
+      - name: controller
+        label: cloud-centos-9-stream-tripleo-vexxhost
+      - name: crc
+        label: coreos-crc-extracted-2-19-0-xxl
+
+- nodeset:
+    name: stf-crc_extracted-ocp413
+    nodes:
+      - name: controller
+        label: cloud-centos-9-stream-tripleo-vexxhost
+      - name: crc
+        label: coreos-crc-extracted-2-28-0-xxl
+
+- nodeset:
+    name: stf-crc_extracted-ocp414
+    nodes:
+      - name: controller
+        label: cloud-centos-9-stream-tripleo-vexxhost
+      - name: crc
+        label: coreos-crc-extracted-2-30-0-xxl
+
+# Based on the 2-node job cookbook at https://github.com/openstack-k8s-operators/ci-framework/blob/main/docs/source/cookbooks/zuul-job-nodeset.md
+- job:
+    name: stf-base-2node
+    parent: podified-multinode-edpm-deployment-crc
+    abstract: true
+    required-projects:
+      - name: github.com/openstack-k8s-operators/dataplane-operator
+        override-checkout: main
+      - name: github.com/openstack-k8s-operators/infra-operator
+        override-checkout: main
+      - name: github.com/openstack-k8s-operators/openstack-operator
+        override-checkout: main
+      - name: github.com/openstack-k8s-operators/openstack-must-gather
+        override-checkout: main
+    pre-run:
+      - ci/pre-2node.yml
+    vars:
+      cifmw_deploy_edpm: false
+      podified_validation: true
+      cifmw_run_tests: false
+    extra-vars:
+      crc_ci_bootstrap_networking:
+        networks:
+          default:
+            range: 192.168.122.0/24
+            mtu: 1500
+          internal-api:
+            vlan: 20
+            range: 172.17.0.0/24
+          storage:
+            vlan: 21
+            range: 172.18.0.0/24
+          tenant:
+            vlan: 22
+            range: 172.19.0.0/24
+        instances:
+          controller:
+            networks:
+              default:
+                ip: 192.168.122.11
+          crc:
+            networks:
+              default:
+                ip: 192.168.122.10
+              internal-api:
+                ip: 172.17.0.5
+              storage:
+                ip: 172.18.0.5
+              tenant:
+                ip: 172.19.0.5
+
+- job:
+    name: stf-base
+    # defined in: https://review.rdoproject.org/cgit/config/tree/zuul.d/_jobs-crc.yaml
+    parent: stf-base-2node
+    abstract: true
+    description: |
+      Run the stf-run-ci role, and then test stf
+    roles:  # adds in dependent roles i.e. put it in the role path
+      - zuul: github.com/openstack-k8s-operators/ci-framework
+    # These are the additional repos that zuul will clone
+    required-projects:
+      - name: openstack-k8s-operators/ci-framework
+        override-checkout: main
+      - name: github.com/infrawatch/service-telemetry-operator
+      - name: github.com/infrawatch/smart-gateway-operator
+      - name: github.com/infrawatch/sg-bridge
+      - name: github.com/infrawatch/sg-core
+      - name: github.com/infrawatch/prometheus-webhook-snmp
```

Comment:
@elfiesmelfie these don't need overrides for `stable-1.5` do they?

CC: @mgirgisf

Reply:
I think we need to add override-checkout to point at the proposed-release branches for the repos in infrawatch.

Note: except  - name: github.com/infrawatch/service-telemetry-operator this don't need to be override to be able to test the changes in this branch

Reply:
I am not 100% sure. They may need an override for a while, until zuul is merged into stable-1.5.

Reply:
There are additonal comments on this conversation below on Chris's proposed patch (which I unnecessarily updated).

I was over thinking this earlier, sorry for all the confusion.

TL;DR: this can stay as-is, but we need to use Depends-On to point at the open PRs until they merge.